docs/actions-new.md

```diff
@@ -0,0 +1,247 @@
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+OpenWhisk supports [several languages and runtimes](actions.md#languages-and-runtimes) but
+there may be other languages or runtimes that are important for your organization, for which
+you want tighter integration with the platform. Adding a new language, or runtime with bespoke
+packages and third party dependencies is the same from the OpenWhisk platform which is language
```

Comment:
Something off in this sentence.  Here's an attempt to maybe fix it..

> Adding a new language, or runtime with bespoke packages and third party dependencies, is the same as a supported language from the perspective of the OpenWhisk platform which is language

Reply:
Thanks for the feedback - I attempted a rewording in the last commit.